namer/namer.go

```diff
@@ -19,14 +19,16 @@ package namer
 var consonants = "bcdfghjklmnpqrstvwxyz"
 
 type Namer struct {
-	// to add any exceptions to look up
+	// use this to add any exceptions to look up
 	Exceptions map[string]string
-	// Use this to either convert everything to lowercase/upper case
+	// TODO: Use this to either convert everything to lowercase/upper case
```

Comment:
Why TODO?

Reply:
My bad. Not applicable here. It may be useful for informer gen later.